src/test/codegen/repr-u128.rs

```diff
@@ -0,0 +1,17 @@
+// compile-flags: --emit=llvm-ir -C debuginfo=2
+#![feature(repr128)]
+
+#[repr(u128)]
+pub enum Foo {
+    Lo,
+    Hi = 1 << 64,
```

Comment:
Hmm, if you use `42_000_000_000_000_000_000_123`, that should be between `1 << 75` and `1 << 76`, and much easier to spot in the IR.

Reply:
Nevermind, check out the updated example in https://github.com/rust-lang/rust/issues/59509#issue-426755256, it's much closer to `1 << 64`, which makes the truncated value smaller (which I forgot about.

Reply:
Okay, I added `bar`.